packages/trpc/server/routers/loggedInViewer/outOfOffice.handler.ts

```diff
@@ -50,6 +50,14 @@ export const outOfOfficeCreate = async ({ ctx, input }: TBookingRedirect) => {
     const user = await prisma.user.findUnique({
       where: {
         id: input.toTeamUserId,
+        /** You can only create OOO for members of teams you're either ADMIN or OWNER */
+        teams: {
+          some: {
+            userId: ctx.user.id,
+            accepted: true,
+            role: { in: ["OWNER", "ADMIN"] },
+          },
+        },
```

Comment:
You will receive an obfuscated not found error if you try to create entries for other users.